File: HotwireServer/models/Message.py
class Message:
    def __init__(self, data):
        self.id = data[0]
        self.sender_id = data[1]
        self.receiver_id = data[2]
        self.sender_nickname = data[3]
        self.content = data[4]

    def serialize(self):
        return {
            'ID': int(self.id),
            'SenderID': self.sender_id,
            'ReceiverID': self.receiver_id,
            'Nickname': self.sender_nickname,
            'Content': self.content,
        }

    def __str__(self):
        return f"ID: {self.id}, SenderID: {self.sender_id}, ReciverID: {self.receiver_id}, Nickname: {self.sender_nickname}, Content{self.content}"

File: Hotwire/Model/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows;

namespace Hotwire.Model
{
    public class Message
    {
        public int ID { get; set; }
        public int SenderID { get; set; }
        public int ReciverID { get; set; }
        public string Nickname { get; set; }
        public string Content { get; set; }
        public HorizontalAlignment Aligment{ get; set; } 
    }
}
